tools/docker-compose-minikube/minikube/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Create _sources directory
+  file:
+    path: "{{ sources_dest }}"
+    state: 'directory'
+    mode: '0700'
+
```

Comment:
need to pip install the openshift module

Reply:
Yes, it's required however I did not include on the playbook because we will not know if the devel wants to use a venv or global python, so left for the devel to hand the python stuff, but if we think installing is important, we can add it